index.html

```diff
@@ -47,11 +47,173 @@ <h1 class="h-title">I’m ROBERTO. Glad to see you!</h1>
+      </p>
+
+      <ul>
+          <li class="tag">
+              <a href="">HTML</a>
+          </li>
+          <li class="tag">
+              <a href="" >CSS</a>
+          </li>
+          <li class="tag">
+              <a href="">JavaScript</a>
+          </li>
+      </ul>
+
+      <a href="#" class="button project">See Project</a>
+       </div>
+
       
-     
-      <section>
 
-      </section>
+  </section>
+  <section  class="card works">
+
+    <img src="img/Snapshoot1.png" alt="a_screentshot" class="pic">
+
+    <div class="text-card">
+      <h2 class="titleh2"> Multi-Post Stories</h2>
+        <ul class="canopy">
+          <li class="cano"><a href="#" class="can">CANOPY</a></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+
+    <p class="texto">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+
+    <ul>
+        <li class="tag">
+            <a href="">HTML</a>
+        </li>
+        <li class="tag">
+            <a href="" >CSS</a>
+        </li>
+        <li class="tag">
+            <a href="">JavaScript</a>
+        </li>
+    </ul>
+
+    <a href="#" class="button project">See Project</a>
+     </div>
+
+    
+
+</section>
+<section  class="card works">
+
+  <img src="img/Snapshoot2.png" alt="a_screentshot" class="pic">
+
+  <div class="text-card">
+      <h2 class="titleh2">Tonic</h2>
+      <ul class="canopy">
+        <li class="cano"><a href="#" class="can">CANOPY</a></li>
+        <li><a href="#">Back End Dev</a></li>
+        <li><a href="#">2015</a></li>
+      </ul>
+
+  <p class="text">
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+
+  <ul>
+      <li class="tag">
+          <a href="">HTML</a>
+      </li>
+      <li class="tag">
+          <a href="" >CSS</a>
+      </li>
+      <li class="tag">
+          <a href="">JavaScript</a>
+      </li>
+  </ul>
+
+  <a href="#" class="button project">See Project</a>
+   </div>
+
+  
+
+</section>
+<section  class="card works">
+
+  <img src="img/Snapshoot3.png" alt="a_screentshot" class="pic">
+
+  <div class="text-card">
+    <h2 class="titleh2"> Multi-Post Stories</h2>
+      <ul class="canopy">
+        <li class="cano"><a href="#" class="can">CANOPY</a></li>
+        <li><a href="#">Back End Dev</a></li>
+        <li><a href="#">2015</a></li>
+      </ul>
+
+  <p class="text">
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+
+  <ul>
+      <li class="tag">
+          <a href="">HTML</a>
+      </li>
+      <li class="tag">
+          <a href="" >CSS</a>
+      </li>
+      <li class="tag">
+          <a href="">JavaScript</a>
+      </li>
+  </ul>
+
+  <a href="#" class="button project">See Project</a>
+   </div>
+
+  
+
+</section>
+  <section class="myself-section">
+    <div class="title">
+        <h1 class="h-title">About Myself</h1>
```

Comment:
- [ ] The `About` section looks good. To make it look exactly like the Figma design, I will suggest that you ensure that the text `About Myself` breaks onto a second line as seen in the design :+1: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/236205532-789ab7f7-976e-406f-a012-95636c499c37.png) | ![image](https://user-images.githubusercontent.com/102699958/236205644-425dcc6b-faaa-4f57-ba1e-b69d3c0c8031.png) |